src/ConfigElement/Operator/Concatenator.php

```diff
@@ -46,8 +47,10 @@ public function getLabeledValue($object)
             $value = $c->getLabeledValue($object) ? $c->getLabeledValue($object)->value : null;
 
             if (!$hasValue) {
-                if (!empty($value) || ((method_exists($value, 'isEmpty') && !$value->isEmpty()))) {
-                    $hasValue = true;
+                if (is_object($value) && method_exists($value, 'isEmpty')) {
```

Comment:
we already fixed this on 5.x branch so let's cherry pick this [change](https://github.com/pimcore/output-data-config-toolkit/pull/58/files#diff-fb6bfa7ad67fc1d05d823f70caf41ad1fba728ab3a02e11a21ab1a50633474eeR49) on 4.1 branch. could you please revert it?